protx/utils/maltreatment.py

```diff
@@ -6,7 +6,6 @@
 
 db_name = '/protx-data/cooks.db'
 
-# TODO.  this should be a mapping that doesn't use display names https://jira.tacc.utexas.edu/browse/COOKS-336
 maltrt_palette = {
```

Comment:
Same issue as demographics.
Its a bad code smell to hardcode these styles into the python logic.
We should  create a `styles.conf.py` config file (or something like that) for all visual style and design info to be tracked for use consistently across all the plot components (and the map).
Especially if we collapse the "many views" into a singular view that handles all the use cases for map + plots.

Reply:
good idea 💯   will move to a conf file as suggested.